src/components/molecules/content-header/content-header.test.tsx

```diff
@@ -25,5 +26,7 @@ describe('ContentHeader', () => {
     expect(screen.getByText('X-Men')).toBeInTheDocument();
     expect(screen.getByText('https://doi.org/10.1101/123456')).toBeInTheDocument();
     expect(screen.getByText('title')).toBeInTheDocument();
+    expect(screen.getByText('Copyright information').parentElement).toHaveAttribute('href');
+    expect(screen.getByText('Copyright information').parentElement?.getAttribute('href')).toStrictEqual('#copyright');
```

Comment:
Is this the right place to put the copyright as it's content header and there is no copyright here?

Reply:
I changed the test to be more generic and now it only checks if the copyright is in the document when it's passed as a parameter.